tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -716,3 +728,59 @@ Check And Uninstall Operator In Openshift
         END
     END
     Close All Browsers
+
+Validate URls
+    [Documentation]   Validates  urls
+    [Arguments]     ${urls}
+    FOR  ${url}   IN   @{urls}
+        ${status}=          Check HTTP Status Code    link_to_check=${url}
+        Log To Console     ${url} gets status code ${status}
+    END
+
+Get Urls and validation of quickStrat Tiles
+    [Documentation]     Clicks on the side window and  validate all Url
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    @{href_list}=       Evaluate           [item.get_attribute('href') for item in $element_list]
+    IF  ${href_list}
+        Validate URls   ${href_list}
+    END
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+            # Sometimes it's not able to click side window button so we are checking for false
+            ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+            IF  ${status} == False
+                Click Button    Next
+            END
+
+            FOR    ${counter}    IN RANGE    5
+                Press Keys    NONE    TAB
+            END
+            Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
```

Comment:
I think you don't need these lines because the TC wants only to check the external links